### PR TITLE
Start replacing Docker with Podman

### PR DESCRIPTION
**Pull request by machadovilaca**
Docker updated its terms of use to require paid subscriptions for almost all users. We still require Docker tools for building and testing. This PR starts replacing the usage of Docker for Podman, a free open-source alternative, in order to reduce unnecessary subscription costs.

Right now, since some testing lanes use images that do not support podman usage yet, we are keeping Docker compatibility in some scripts.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
Replace Docker commands with Podman
```

### CODE REVIEW

File: automation/nightly/test-nightly-build.sh [hunk 1]
```diff
@@ -31,9 +33,9 @@ go mod vendor
 build_date="$(date +%Y%m%d)"
 export IMAGE_REGISTRY=quay.io
 export IMAGE_TAG="nb_${build_date}_$(git show -s --format=%h)"
-export DOCKER_PREFIX=kubevirtci
-TEMP_OPERATOR_IMAGE=${DOCKER_PREFIX}/hyperconverged-cluster-operator
-TEMP_WEBHOOK_IMAGE=${DOCKER_PREFIX}/hyperconverged-cluster-webhook
+export CONTAINER_PREFIX=kubevirtci
```
Comment (nunnatsa) [inline automation/nightly/test-nightly-build.sh:36]: Please consider:
```suggestion
export IMAGE_PREFIX=kubevirtci
```

**Comment by machadovilaca** [comment]
@nunnatsa updated

**Pull request merged**